### PR TITLE
UX-376 - Update to alton theme

### PR DESCRIPTION
**Pull request by nickphillipps**
#### What does this PR do? (please provide any background)
- Add styling for altons secure nav bar
- Tweaks to payment summary for Alton

#### What tests does this PR have?
None

#### How can this be tested?
Checkout this branch on both this repo `brands` and pattern library
in pattern library run `npm link ../brands` then run `npm start`
Navigate to page > Shortbreaks payment

#### Screenshots / Screencast
screencapture-localhost-3000-pages-shortbreaks-payment-index-html-1460623282815
screencapture-localhost-3000-pages-shortbreaks-payment-index-html-1460623260078

#### What gif best describes how you feel about this work?
tumblr_mkxzufnkl71rjffk3o1_400

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: brands/alton-towers/src/less/_navbar.less [hunk 1]
```diff
@@ -76,3 +76,28 @@
...
+  .container-fluid .navbar-brand{
+    margin-top: 15px;
+  }
+  .contact-link{
+    color: white;
+    font-family: @headings-font-family;
+  }
+}
+
+.navbar-right{
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/_navbar.less]: `navbar-right` should already float right?

**Comment by jodiedoubleday** [comment]
I seem to get this navbar...
screen shot 2016-04-15 at 13 15 16

**Comment by nickphillipps** [comment]
@jodiedoubleday With the nav bar, you're seeing the new markup in pattern library but without the new less in the brands pr

**Comment by jodiedoubleday** [comment]
@nickphillipps I've NPM linked the two, ill try again and let you know

**Comment by jodiedoubleday** [comment]
All working now, header works perfectly, though doesn't look like the existing alton header or alton towers breaks header, is this okay?

File: brands/alton-towers/src/less/theme.less [hunk 2]
```diff
@@ -15,3 +15,8 @@
 
 // Custom
 @import "_fonts";
+@import "_summary";
+
+body{
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/theme.less]: Maybe should consider making this a variable in the future?

Comment (nickphillipps) [inline brands/alton-towers/src/less/theme.less]: @jodiedoubleday I agree, with the merlin brands they will all need a background image

**Comment by nickphillipps** [comment]
@jodiedoubleday On the nav, theres work being done on it at the mo to match up with the styling on here

**Comment by jodiedoubleday** [comment]
Yup all good, my only concern is using the class `.navbar-secure` but if that already exists I'm happy.

**Comment by nassimdonald** [comment]
👍

File: brands/alton-towers/src/less/_navbar.less [hunk 3]
```diff
@@ -76,3 +76,24 @@
     }
   }
 }
+
+.navbar-secure{
```
Comment (nassimdonald) [inline brands/alton-towers/src/less/_navbar.less]: Is there a reason you're not just using the base `.navbar` class? Is this navbar just for the payment page?

File: brands/alton-towers/src/less/_navbar.less [hunk 4]
```diff
@@ -76,3 +76,24 @@
...
+  background:@body-bg;
+  max-width: none;
+  margin: 0 0 40px;
+  border-bottom: solid 2px @brand-secondary;
+  box-shadow: inset 0 -5px 20px 0px darken(@body-bg, 15%);
+  .container-fluid .navbar-brand{
+    margin-top: 15px;
+  }
+  .contact-link{
+    color: white;
```
Comment (nassimdonald) [inline brands/alton-towers/src/less/_navbar.less]: You could use `@navbar-default-color` for this.

File: brands/alton-towers/src/less/_type.less [hunk 5]
```diff
@@ -25,3 +25,12 @@ h1, h2, h3, h4,
   h3, .h3 {font-size: 1.75em;}    // 28px
   h4, .h4 {font-size: 1.5em;}     // 24px
 }
+
+@dl-horizontal-offset: 40%;
+@media (min-width: @screen-sm) {
```
Comment (nassimdonald) [inline brands/alton-towers/src/less/_type.less]: You may want to change this breakpoint to `@screen-md-min`, as that is when the `<dd>` adapts to display the `<dt>` inline. Currently, you are causing the `<dt>` to wrap unnecessarily at certain screen sizes…
image

File: brands/alton-towers/src/less/_navbar.less (hunk 3 above)
Comment (nickphillipps) [inline brands/alton-towers/src/less/_navbar.less]: Currently on theme we have two seperate nav bars, on on seo side and one on secure side. Figured i'd use that to hook onto for the styling

**Pull request merged**